.ort.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2023 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+curations:
+  packages:
+  - id: "PyPI::identify:2.5.24"
+    curations:
+      comment: "Proper license is defined in package repository https://pypi.org/project/identify/"
+      concluded_license: "MIT"
+  - id: "Conan::c-ares:1.19.0"
+    curations:
+      comment: "Proper license is defined in package repository https://conan.io/center/c-ares"
+      concluded_license: "MIT"
+  - id: "Conan::cpr:1.9.3"
+    curations:
+      comment: "Proper license is defined in package repository https://conan.io/center/cpr"
+      concluded_license: "MIT"
+  - id: "Conan::googleapis:cci.20221108"
+    curations:
+      comment: "Proper license is defined in package repository https://conan.io/center/googleapis?version=cci.20221108"
+      concluded_license: "Apache-2.0"
+  - id: "Conan::libcurl:7.87.0"
+    curations:
+      comment: "Proper license is defined in package repository https://conan.io/center/libcurl?version=7.87.0"
+      concluded_license: "curl"
+  - id: "Conan::openssl:1.1.1t"
+    curations:
+      comment: "Proper license is defined in package repository https://conan.io/center/openssl?version=1.1.1t"
+      concluded_license: "OpenSSL"
+  - id: "Conan::openssl:1.1.1t"
```

Comment:
Is it OpenSSL package or shall be something else? Probably you meant paho-mqtt-c

Reply:
yes of course I meant paho...